src/nft/components/explore/Explore.css.ts

```diff
@@ -106,3 +106,68 @@ export const carouselIndicator = sprinkles({
   borderRadius: 'round',
   display: 'inline-block',
 })
+
+/* Value Prop Styles */
+export const valuePropWrap = style([
+  {},
+  sprinkles({
+    width: 'full',
+    borderWidth: '1px',
+    borderStyle: 'solid',
+    borderColor: 'grey700',
+    borderRadius: '12',
+    paddingLeft: '16',
+    paddingRight: '16',
+    marginTop: '60',
+    position: 'relative',
+  }),
+])
+
+export const valuePropOverlay = style([
+  {
+    opacity: '0.85',
+    height: '135px',
+    '@media': {
+      [`(min-width: ${breakpoints.lg}px)`]: {
+        opacity: '0.2',
```

Comment:
opacities have now been added to sprinkles

Reply:
nice - thats very helpful